MultiSource/Benchmarks/Ptrdist/ft/ft.c

```diff
@@ -67,7 +67,7 @@ _Ptr<Vertices>  MST(_Ptr<Vertices>  graph);
 int debug = 1;
 
 int
-main(int argc, _Array_ptr<_Ptr<char>> argv )
+main(int argc, _Array_ptr<const char*> argv : count(argc) )
```

Comment:
Ah, I have only just found out that the automated converter can currently only output `_Ptr<T>` types, and will only use the underscore version. If it's the case you used the converter, leave them as `_Ptr<T>`, when we improve the converter it will manage to output `ptr<T>`.